src/satctl.c

```diff
@@ -57,21 +57,49 @@ void usage(void)
 	printf("Copyright (c) 2014 Satlab ApS <satlab@satlab.com>\n");
 	printf("\n");
 	printf("Options:\n");
-	printf(" -c INTERFACE,\tUse INTERFACE as CAN interface\n");
-	printf(" -u INTERFACE,\tUse INTERFACE as UART interface\n");
-	printf(" -b BAUD,\tUART buad rate\n");
-	printf(" -n NODE\tUse NODE as own CSP address\n");
-	printf(" -r UDP_CONFIG\tUDP configuration string, encapsulate in brackets: \"<lport> <peer ip> <rport>\" (supports multiple) \n");
-	printf(" -z ZMQ_IP\tIP of zmqproxy node (supports multiple)\n");
-	printf(" -p\t\tSetup prometheus node\n");
-	printf(" -R RTABLE\tOverride rtable with this string\n");
-	printf(" -h\t\tPrint this help and exit\n");
+	printf(" -c INTERFACE,\t\t\tUse INTERFACE as CAN interface\n");
+	printf(" -u INTERFACE,\t\t\tUse INTERFACE as UART interface\n");
+	printf(" -b BAUD,\t\t\tUART buad rate\n");
+	printf(" -n NODE\t\t\tUse NODE as own CSP address\n");
+	printf(" -r UDP_CONFIG\t\t\tUDP configuration string, encapsulate in brackets: \"<lport> <peer ip> <rport>\" (supports multiple) \n");
```

Comment:
Maybe we should use the same syntax as here (or modify this syntax) - now we have two in the same program.

Reply:
Which syntax do you mean? I didn't get it.

Reply:
Just disregard this.

Reply:
Ah do you mean the UDP config? Hmm I can take a look at how it was parsed.

Reply:
Just update the help message and it should be fine.